idl/rrdb.thrift

```diff
@@ -279,11 +279,15 @@ struct get_scanner_request
     11:optional bool    validate_partition_hash;
     12:optional bool    return_expire_ts;
     13:optional bool full_scan; // true means client want to build 'full scan' context with the server side, false otherwise
+    14:optional filter_type value_filter_type;
+    15:optional dsn.blob    value_filter_pattern;
+    16:optional bool    only_return_count = false;
 }
 
 struct scan_request
 {
     1:i64           context_id;
+    2:optional bool only_return_count;
```

Comment:
the `get_scanner_request ` has a `only_return_count` field, why add another one to `scan_request`?